Makefile

```diff
@@ -275,7 +281,7 @@ generated_definitions/c/riscv.c: $(SAIL_SRCS) $(SAIL_RISCV_MODEL_DIR)/main.sail
 
 generated_definitions/c/riscv_model_%.c: $(SAIL_SRCS) $(SAIL_RISCV_MODEL_DIR)/main.sail Makefile
 	mkdir -p generated_definitions/c
-	$(SAIL) $(SAIL_FLAGS) -O -Oconstant_fold -memo_z3 -c -c_include riscv_prelude.h -c_include riscv_platform.h -c_no_main $(SAIL_SRCS) $(SAIL_RISCV_MODEL_DIR)/main.sail -o $(basename $@)
+	$(SAIL) $(SAIL_FLAGS) -O -Oconstant_fold -memo_z3 -c -c_include riscv_prelude.h -c_include riscv_platform.h -c_no_main $(SAIL_SRCS) $(SAIL_RISCV_MODEL_DIR)/main.sail -o $(basename $@) > generated_definitions/c/all_branches
```

Comment:
This doesn't seem like a great interface from sail... stdout really isn't meant for this.

Reply:
If you want to change it I don't mind. It was just the most braindead simple solution rather than passing an extra file handle to everywhere it might be needed.

Reply:
Thinking about it changing `val branch_coverage : bool` into `val branch_coverage : out_channel option` in the ANF -> IR config module is probably the best way to go. I won't have time to do this before our PLDI workshop next week but it is probably a pretty easy change.